CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ librdkafka v1.9.3 is a maintenance release:
 ## Enhancements
 
  * Bundled zlib upgraded to version 1.2.13.
+ * The C++ API no longer returns strings "by const value," which enables better move optimization in callers.
```

Comment:
```suggestion
 * The C++ API no longer returns strings "by const value" which enables better move optimization in callers.
```

Reply:
Well, we want to make it (grammatically) clear that it's the _removal_ of "return by const value" that enables the better optimization. We're not removing something _which enables optimization._ We're removing something, _which enables optimization._

Could change to `* To enable better move optimization in callers, the C++ API no longer returns strings "by const value."` if you like that better.

Reply:
Or simply remove the quotes, I think that's probably the best option.